payments/vipps/index.md

```diff
@@ -18,750 +18,63 @@ sidebar:
 
-{:.code-header}
-**Response**
+The payment request will appear in the app and can be confirmed by the consumer.
+If you are using the redirect option, the consumer will be redirected back to
+the merchant's site. 
 
-```http
-HTTP/1.1 200 OK
-Content-Type: application/json
+## Good To Know
 
-{
-   "payment": {
-       "id": "/psp/vipps/payments/{{ page.paymentId }}",
-       "number": 72100003079,
-       "created": "2018-09-05T14:18:44.4259255Z",
-       "instrument": "Vipps",
-       "operation": "Purchase",
-       "intent": "Authorization",
-       "state": "Ready",
-       "currency": "NOK",
-       "prices": {
-           "id": "/psp/vipps/payments/{{ page.paymentId }}/prices"
-        },
-       "amount": 0,
-       "description": "Vipps Test",
-       "payerReference": "AB1536157124",
-       "initiatingSystemUserAgent": "PostmanRuntime/7.2.0",
-       "userAgent": "Mozilla/5.0 weeeeee",
-       "language": "nb-NO",
-       "urls": {
-           "id": "/psp/vipps/payments/{{ page.paymentId }}/urls"
-        },
-       "payeeInfo": {
-           "id": "/psp/vipps/payments/{{ page.paymentId }}/payeeinfo"
-        },
-       "metadata": {
-           "id": "/psp/vipps/payments/{{ page.paymentId }}/metadata"
-        }
-    },
-   "operations": [
-        {
-           "method": "PATCH",
-           "href": "{{ page.apiUrl }}/psp/vipps/payments/{{ page.paymentId }}",
-           "rel": "update-payment-abort"
-        },
-        {
-           "method": "GET",
-           "href": "{{ page.frontEndUrl }}/vipps/payments/authorize/afccf3d0016340620756d5ff3e08f69b555fbe2e45ca71f4bd159ebdb0f00065",
-           "rel": "redirect-authorization"
-        }
-    ]
-}
-```
+### Payment Type
 
-### Purchase
+Vipps is one of the instruments using two-phase payments. The `authorization` is
+done when the consumer successfully confirms in the app, and the `abort`,
+`cancel`, `capture` or `reversal` is done by the merchant at a later time. Read
+more about the different operations [here][after-payment] and the payment
+resource [here][payment-resource].
 
-Posting a payment (operation `purchase`) returns the options of aborting the
-payment altogether or creating an authorization transaction through the
-redirect-authorization hyperlink.
+### Settlement
 
-{:.code-header}
-**Request**
+Vipps transactions are handled, processed and settled as card transactions in
+our system. They are, however, tagged as Vipps transactions and have their own
+acquirer agreement, so the two payment instruments are settled separately.
 
-```js
-{
-   "payment": {
-       "operation": "Purchase"
-    }
-}
-```
+### 3D Secure
 
-### Operations
+As Vipps transactions are processed as card transactions, you will also need a
+3D Secure agreement. If you already have a 3D Secure agreement used in the card
+contract setup, this can be re-used.
```

Comment:
Perhaps we should say something about having a 3-D Secure agreement not being the same as being 3-D Secure Compliant?